tools/build_pak_files.py

```diff
@@ -15,26 +15,20 @@
 if __name__ == '__main__':
 
     parser = argparse.ArgumentParser()
-    parser.add_argument("--unreal_engine_dir", required=True)
-    parser.add_argument("--perforce_content_dir", required=True)
+    parser.add_argument("--unreal_engine_dir", required=True)    
     parser.add_argument("--output_dir", required=True)
     parser.add_argument("--version_tag", required=True)
+    parser.add_argument("--skip_create_symlinks", action="store_true")
+    parser.add_argument("--perforce_content_dir")
     parser.add_argument("--scene_ids")
     args = parser.parse_args()
-    
-    assert os.path.exists(args.unreal_engine_dir)
-    assert os.path.exists(args.perforce_content_dir)
 
     unreal_project_dir          = os.path.realpath(os.path.join(os.path.dirname(__file__), "..", "cpp", "unreal_projects", "SpearSim"))
     uproject                    = os.path.realpath(os.path.join(unreal_project_dir, "SpearSim.uproject"))
     unreal_project_content_dir  = os.path.realpath(os.path.join(unreal_project_dir, "Content"))
```

Comment:
let's remove the extra whitespace before the `=` now that we have removed `perforce_content_shared_dir`

Reply:
Done